src/nft/components/collection/Sweep.tsx

```diff
@@ -211,52 +209,12 @@ export const Sweep = ({ contractAddress, collectionStats, minPrice, maxPrice, sh
     }
   }
 
-  const { data: collectionAssets, isFetched: isCollectionAssetsFetched } = useQuery(
-    ['sweepAssets', getSweepFetcherParams('others')],
-    () => fetchSweep(getSweepFetcherParams('others')),
-    {
-      refetchOnWindowFocus: false,
-      refetchOnMount: false,
-      refetchOnReconnect: false,
-    }
-  )
-
-  const { data: nftxCollectionAssets, isFetched: isNftxCollectionAssetsFetched } = useQuery(
-    ['nftxSweepAssets', collectionStats, getSweepFetcherParams(Markets.NFTX)],
-    () =>
-      collectionStats.marketplaceCount?.some(
-        (marketStat) => marketStat.marketplace === Markets.NFTX && marketStat.count > 0
-      )
-        ? fetchSweep(getSweepFetcherParams(Markets.NFTX))
-        : [],
-
-    {
-      refetchOnWindowFocus: false,
-      refetchOnMount: false,
-      refetchOnReconnect: false,
-    }
-  )
-
-  const { data: nft20CollectionAssets, isFetched: isNft20CollectionAssetsFetched } = useQuery(
-    ['nft20SweepAssets', getSweepFetcherParams(Markets.NFT20)],
-    () =>
-      collectionStats.marketplaceCount?.some(
-        (marketStat) => marketStat.marketplace === Markets.NFT20 && marketStat.count > 0
-      )
-        ? fetchSweep(getSweepFetcherParams(Markets.NFT20))
-        : [],
-
-    {
-      refetchOnWindowFocus: false,
-      refetchOnMount: false,
-      refetchOnReconnect: false,
-    }
-  )
-
-  const allAssetsFetched = isCollectionAssetsFetched && isNftxCollectionAssetsFetched && isNft20CollectionAssetsFetched
+  const collectionAssets = useSweepAssetsQuery(getSweepFetcherParams('others'))
```

Comment:
can you name this otherCollectionAssets?. also i think Others should be in the Markets enum

Reply:
There's scoped work to include new marketplaces from gql in sweep which at that point I think it would be worth looking into a smarter handling of if assets are pooled or not. I intentionally didn't want to alter that behaviour in this PR to avoid scope creep.